BaseSpace.SDK.Tests/Integration/BaseIntegrationTest.cs

```diff
@@ -42,15 +45,25 @@ protected ILog Log
             }
         }
 
+
+        public static string GetConfigValue(string key)
+        {
+            var module = System.Reflection.Assembly.GetExecutingAssembly().GetName().Name + ".dll";
+            var config = ConfigurationManager.OpenExeConfiguration(module);
```

Comment:
Maybe config should be a private variable that is set in the constructor or you can also put it into a fixture so lines 51-52 is only executed once instead of each time you want to get a config setting value.